documentation/index.html

```diff
@@ -409,11 +409,167 @@ <h1>/get</h1>
    "day":"1"
 }
 
-			</pre>
+            </pre>
 
-		</div>
+        </div>
+
+        <div class="ep">
+            <span class="method">GET</span><span class="root">api.cinelist.co.uk</span><span class="path">/get/times/many/:venueIDs?day=&#60;INT&#62;</span>
+            <p>Get the film times for the provided list of cinemas. The list of cinemas should be separated by commas. The day query parameter is an offset to get times for a day other than today.</p>
+
+            <pre class="result">
+// Request: http://www.cinelist.co.uk/get/times/many/10681,7517
```

Comment:
This line should read:
`// Request: http://api.cinelist.co.uk/get/times/many/10681,7517` (`api.cinelist.co.uk`)instead of `www.cinelist.co.uk`.